numpy/core/src/common/simd/intdiv.h

```diff
@@ -89,7 +89,9 @@ NPY_FINLINE unsigned npyv__bitscan_revnz_u32(npy_uint32 a)
     unsigned long rl;
     (void)_BitScanReverse(&rl, (unsigned long)a);
     r = (unsigned)rl;
-#elif defined(NPY_HAVE_SSE2) && (defined(__GNUC__) || defined(__clang__) || defined(__INTEL_COMPILER))
+
+#elif defined(NPY_HAVE_SSE2) && (defined(__GNUC__) || defined(__clang__) || defined(__INTEL_COMPILER)) \
```

Comment:
So the NPY_HAVE_SSE2 is redundant now. Does WASM have `__builtin_clz` (below)
```suggestion
#elif (defined(__GNUC__) || defined(__clang__) || defined(__INTEL_COMPILER)) \
```

Reply:
Isn't `NPY_HAVE_SSE2` also gated on whether the user passed `--disable-optimizations`?

Reply:
Yes wasm has __builtin_clz

Reply:
I think if you can do the bsr via a compiler intrinsic rather than with asm it would work in wasm too.

Reply:
Maybe in a future PR... let's just get this in for now.